opencog/query/QueryModule.cc

```diff
@@ -12,16 +12,18 @@ using namespace opencog;
 
 DECLARE_MODULE(QueryModule);
 
-QueryModule::QueryModule(CogServer& cs) : Module(cs), _pat(NULL)
+QueryModule::QueryModule(CogServer& cs) : Module(cs), _pat(NULL), _fpat(NULL)
 {
 }
 
 QueryModule::~QueryModule()
 {
 	delete _pat;
+	delete _fpat;
 }
 
 void QueryModule::init(void)
 {
 	_pat = new PatternSCM();
+	_fpat = new FuzzyPatternSCM();
```

Comment:
This is the wrong place to add this, it should be added to PatternSCM instead.  You won't need class FuzzyPatternSCM either.  Just add a slightly modified version of find_approximate_matches to it, something like

```
_binders.push_back(new PatternWrap(find_approximate_matches, "cog-fuzzy-match"));
```

can change the signature on find_approximate_matches so that it uses the atomspace that it is given.